src/Estimators/tests/test_StructureFactorEstimator.h

```diff
@@ -0,0 +1,659 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2025 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef TEST_STRUCTUREFACTORESTIMATOR_H
+#define TEST_STRUCTUREFACTORESTIMATOR_H
+#include "StructureFactorEstimator.h"
+
+namespace qmcplusplus
+{
+namespace testing
+{
+class StructureFactorAccess
+{
+  using Real = StructureFactorEstimator::Real;
+
+public:
+  const Vector<Real>& getSKElecElec(const StructureFactorEstimator& sfe) const { return sfe.getSKElecElec(); }
+  const Vector<std::complex<Real>>& getRhoKElec(const StructureFactorEstimator& sfe) const { return sfe.getRhoKElec(); }
+};
+} // namespace testing
+
+/// Provides the canned test data for testing
+class StructureFactorTests
```

Comment:
Just wondering why this test has a header file. Will it be included in other places?

Reply:
The templating of the canned data requires quite a particular order and the .cpp was becoming too much for some of the tooling I use, including clang format.